src/comparison.py

```diff
@@ -5,33 +5,68 @@
 CAT_papers = pd.read_csv("CAT_datasets.tsv", sep="\t", index_col="id")
 CAT_skids = pd.read_csv("CAT_skids.tsv", sep="\t")
```

Comment:
In future - better to have code that passes tables rather than relying on standard filenames.